WordPress/gutenberg
# Update the template preview menu item text in the Template option

## What?

This is a text change in the Template option in the Summary panel in the Page settings panel.
The text in the menu item that toggles the visibility of the template is changed from "Template preview" to "Show template".

## Why?

The current text can be confusing, see https://github.com/WordPress/gutenberg/issues/56269

## Testing Instructions




Go to Editor > Appearance > Pages
Select a page to edit.
Open the Page tab in the settings panel.
Open the Summary panel.
Click on the template name next  to "Template"
A popup with a menu opens. Confirm that the last item in the menu is "Show template".

## Screenshots or screencast 
After:
<img width="364" alt="The last menu item in the Template option panel has the text Show template'" src="https://github.com/WordPress/gutenberg/assets/7422055/49e2bdd6-a341-46a7-ba8d-ef4c030e65b2">
